logic/telemetry.go

```diff
@@ -41,6 +42,8 @@ func sendTelemetry() error {
 	if err != nil {
 		return err
 	}
+	// get tenant admin email
+	adminEmail := os.Getenv("NM_EMAIL")
```

Comment:
we should make change in the script not to generate random email, only take input from the user